src/main/java/com/simanavets/tenderflexbackend/entity/Privilege.java

```diff
@@ -0,0 +1,42 @@
+package com.simanavets.tenderflexbackend.entity;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+public class Privilege {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    @Enumerated(EnumType.STRING)
+    private PrivilegeName name;
+
+    @ManyToMany(mappedBy = "privileges")
```

Comment:
Using this you need also provide which field you will be join with (@JoinColumn). Please read about OneToMany and ManyToMany first, because this will not be work.